inspirehep/base/templates/search/results.html

```diff
@@ -148,7 +148,7 @@
     <div class="panel-body" >
       <div class="row">
         <div class="col-md-9">
-          <span class="text-left {{ request.args.get('cc')|lower() }}" id="clear-facets-message">{{ pagination.total_count }} of {{ request.args.get('p')|number_of_search_results(request.args.get('cc')) }} results match the filters you selected.</span><br/>
+          <span class="text-left {{ request.args.get('cc')|lower() }}" id="clear-facets-message">{{ pagination.total_count }} of {{ request.args.get('p', '')|number_of_search_results(collection.name) }} results match the filters you selected.</span><br/>
```

Comment:
Just in case, we are always sure that the `collection` object is available when the template renders right? Referring to the past issues we've had with the 500 page etc.